pkg/poolcache/poolcache.go

```diff
@@ -22,6 +22,7 @@ import (
 	"fmt"
 
 	ferror "github.com/fission/fission/pkg/error"
+	"k8s.io/apimachinery/pkg/api/resource"
 )
 
```

Comment:
This cache's structure is fairly complex at this point, would be nice to add a high level comment with structure and usage so it is easier on reader

Reply:
Done